user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -98,12 +98,27 @@ The data is encrypted both at rest and in transit.
 
 If [ZRS](#data-location-and-redundancy) is used, STaaS has an expected availability of 99.90%. With [GRS](#data-location-and-redundancy), it has an expected availability of 99.95%. For more information, please contact <sales@skyline.be>.
 
+## TTL
+
+- It is not yet possible to configure time-to-live (TTL) values for STaaS. In the table below, you can find the default TTL values for each data type.
+
+| Data type            | TTL          |
```

Comment:
The data types listed below might not be entirely clear to the user. Would it be possible to put descriptions of the data types rather than these names (e.g. "Real-time trending" instead of "RTTrend")?